os/arch/arm/src/armv7-m/up_mpu.c

```diff
@@ -377,3 +377,40 @@ void up_mpu_set_register(uint32_t *mpu_regs)
 		putreg32(mpu_regs[REG_RASR], MPU_RASR);
 	}
 }
+
+/****************************************************************************
+ * Name: up_mpu_disable_region
+ *
+ * Description:
+ *   Update tcb's mpu register values to disable a mpu region.
+ *   MPU region to be disabled is identified using tcb's mpu registers passed.
+ *   So, if the current MPU h/w settings for this MPU region match to those
+ *   of registers passed, set these new register values to mpu h/w as well.
+ *
+ ****************************************************************************/
+void up_mpu_disable_region(uint32_t *mpu_regs)
+{
+	uint32_t regval;
+
+	/* For kernel task, no operation should be performed. Return immediately */
+	if (!mpu_regs[REG_RBAR]) {
```

Comment:
Could you confirm this does not give SVACE warning for null dereference (I mean null check first)?

Reply:
The way how to check SVACE for this is making a PR in RS6. RS6 has the CI for that.

Reply:
ok..  i will check this soon.

Reply:
SVACE check here is no longer needed as this condition has been removed.